cmd/bbolt/command_check.go

```diff
@@ -4,25 +4,36 @@ import (
 	"fmt"
 
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
 
 	bolt "go.etcd.io/bbolt"
 	"go.etcd.io/bbolt/internal/guts_cli"
 )
 
+type checkPageOptions struct {
+	pageId uint64
+}
+
+func (o *checkPageOptions) AddFlags(fs *pflag.FlagSet) {
+	fs.Uint64VarP(&o.pageId, "pageId", "", o.pageId, "check db integrity starting from the given pageId")
```

Comment:
I wonder whether this flag name might be misleading? what about `from-page`?

similar to how we have it in surgery:
https://github.com/etcd-io/bbolt/blob/main/cmd/bbolt/command_surgery.go#L95

Reply:
`from-page` makes sense!